### PR TITLE
adding two factor gateway to localsettings

### PR DESCRIPTION
**Pull request by calellowitz**
@benrudolph

### CODE REVIEW

File: ansible/roles/commcarehq/templates/localsettings.py.j2 [hunk 1]
```diff
@@ -697,3 +697,11 @@ WS4REDIS_CONNECTION = {
     'host': '{{ localsettings.WS4REDIS_CONNECTION_HOST }}'
 }
 {% endif %}
+
+{% if localsettings.get('TWO_FACTOR_SMS_GATEWAY') %}
+TWO_FACTOR_SMS_GATEWAY = '{{TWO_FACTOR_SMS_GATEWAY}}'
```
Comment (benrudolph) [inline ansible/roles/commcarehq/templates/localsettings.py.j2]: this should be `{{ localsettings.TWO_FACTOR_SMS_GATEWAY }}`

File: ansible/roles/commcarehq/templates/localsettings.py.j2 [hunk 2]
```diff
@@ -697,3 +697,11 @@ WS4REDIS_CONNECTION = {
...
+{% endif %}
+
+{% if localsettings.get('TWO_FACTOR_CALL_GATEWAY') %}
+TWO_FACTOR_CALL_GATEWAY = '{{TWO_FACTOR_CALL_GATEWAY}}'
```
Comment (benrudolph) [inline ansible/roles/commcarehq/templates/localsettings.py.j2]: same here

**Comment by TylerSheffels** [comment]
Looks like you need to merge master to fix the build FYI

**Comment by benrudolph** [comment]
merging in cause this is blocking me deploying other localsettings and this won't actually break the tests

**Pull request merged**